automapper/src/main/java/com/valqueries/automapper/ValqueriesCrudRepositoryBase.java

```diff
@@ -34,7 +32,7 @@
 	protected Class<K> keyType;
 	protected TypeDescriber<T> typeDescriber;
 	protected MappingHelper mappingHelper;
-	private SqlNameFormatter sqlNameFormatter;
+	private final SqlNameFormatter sqlNameFormatter;
 
 	public ValqueriesCrudRepositoryBase(Database database, GenericFactory genericFactory, Class<T> modelType, Class<K> keyType, MappingHelper mappingHelper, SqlNameFormatter sqlNameFormatter) {
```

Comment:
* ValqueriesDataAccessLayer
(also rename the implementations to match the naming scheme)